website/blog/2023-05-30-0.72-metro-package-exports-symlinks.md

```diff
@@ -0,0 +1,172 @@
+- Implement [AggregateError](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/AggregateError) ([9b25](https://github.com/facebook/hermes/commit/9b25a2530eb515f6c4fbd397ae290b6c97c049b2)) that represents several errors wrapped in a single error. Useful for multiple errors like from [`Promise.any()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/any) when all promises passed to it reject.
+
+For users on JSC, these features are already available.
+
+### Moving New Architecture Updates
+
+At this stage, New Architecture updates are most relevant to early adopters of the New Architecture.
+
+To keep updates clear and focused on the target audience, we are moving New Architecture updates in 0.72, and future releases, to the dedicated [working group](https://github.com/reactwg/react-native-new-architecture/discussions). This change will also allow us to share more frequent updates, such as work that ships in our nightlies.
+
+<!-- TODO link -->
+
+You can read the [0.72 updates for the New Architecture here](https://github.com).
+
+Subscribe to the working group GitHub Notifications to stay informed on our progress on the New Architecture.
+
+## Breaking Changes
+
+### Deprecated Component Removals
+
+The following packages have been removed from React Native in 0.72. Please migrate to the recommended community package:
+
+- [Slider](/docs/next/slider) is superseded by [@react-native-community/slider](https://github.com/callstack/react-native-slider/tree/main/package)
+- [DatePickerIOS](/docs/next/datepickerios) is superseded by [@react-native-community/datetimepicker](https://github.com/react-native-datetimepicker/datetimepicker)
+- [ProgressViewIOS](/docs/next/progressviewios) is superseded by [@react-native-community/progress-view](https://github.com/react-native-progress-view/progress-view)
+
+### Package Renames
+
+All packages published from the [`react-native`](https://github.com/facebook/react-native) core repository now live under `react-native/packages`, and are published under the [@react-native npm scope](https://www.npmjs.com/search?q=%40react-native) to ensure clear ownership.
+
+There are no changes to the [react-native](https://www.npmjs.com/package/react-native) package.
+
+Here are all the packages renamed:
+
+- `@react-native-community/eslint-config` to `@react-native/eslint-config`
+- `@react-native-community/eslint-plugin` to `@react-native/eslint-plugin`
+- `@react-native/polyfills` to `@react-native/js-polyfills`
+- `@react-native/normalize-color` to `@react-native/normalize-colors`
+- `@react-native/assets` to `@react-native/assets-registry`
+- `react-native-codegen` to `@react-native/codegen`
+- `react-native-gradle-plugin` to `@react-native/gradle-plugin`
```

Comment:
Should we add a Table here instead? Btw, I don't know how tables are formatted on the website...

![Screenshot 2023-05-30 085644](https://github.com/facebook/react-native-website/assets/55224033/1c0ea2c2-dd58-42fa-a3bd-aca9c75ba83e)

Reply:
Will suggest in a bit. Btw you good with column headings/names above?

Reply:
I've updated! Thanks for the great suggestion!